### ISSUE TITLE
[Vanilla Bug] A group of infantry can't be tasked to enter a Hospital or Armory

### ISSUE THREAD

**Issue by MNOrac** [Vinifera-Developers/Vinifera#59]
### Description:
Infantry can't be group selected and tasked on a Hospital or Armory building. They have to be tediously selected one by one.

### Expected Behaviour:
There is an inconsistency between this behaviour and tasking vehicles to a repair bay, or infantry to an APC. All should allow a group of units to be tasked on it, and the group should wait their turns to enter. The obvious point of comparison is to the RA2 Hospital, which does this.

Given the minutiae of this issue I can see it slipping into Feature Request territory.

**Comment by CCTiberius** [comment]
Yes please, would go such a long way in making Hospitals worthwhile.
